libs/styles/styles.css

```diff
@@ -284,164 +794,305 @@
   line-height: var(--type-body-xxs-lh);
 }
 
-.detail-xl {
-  font-size: var(--type-detail-l-size);
-  line-height: var(--type-detail-l-lh);
-}
+/* Icons Images */
 
-.detail-l {
-  font-size: var(--type-detail-l-size);
-  line-height: var(--type-detail-l-lh);
+img {
+  max-width: 100%;
+  height: auto;
 }
 
-.detail-m {
-  font-size: var(--type-detail-m-size);
-  line-height: var(--type-detail-m-lh);
+svg.icon-milo, img.icon-milo {
+  height: 1em;
+  position: relative;
+  top: .1em;
+  width: auto;
 }
 
-.detail-s {
-  font-size: var(--type-detail-s-size);
-  line-height: var(--type-detail-s-lh);
+.con-button svg.icon-milo {
+  display: inline-block;
+  height: 1em;
+  width: 1em;
 }
 
-.detail-xs {
-  font-size: var(--type-body-xs-size);
-  line-height: var(--type-body-xs-lh);
-  font-weight: 700;
+/* Button Sizes */ 
+.con-button { /* Size M - Secondary Static Black (Default) */
+  --m-button-height: 32px;
+
```

Comment:
I assume the extra line is intentional

Reply:
Yes there is a linting opinion expecting an empty line before variable declarations but we can remove and ignore the linter opinion if necessary.

Reply:
I agree with the Linter's opinion, variables should be clearly separated.